include/aikido/control/Executor.hpp

```diff
@@ -74,7 +74,7 @@ class Executor
   std::set<ExecutorType> getTypes() const;
 
   /// Get list of dofs needed by this Executor
```

Comment:
Can fix the comment. Leaving the function name as-is because it is consistent with DART, see https://dartsim.github.io/dart/v6.8.5/d4/d4a/classdart_1_1dynamics_1_1MetaSkeleton.html